### ISSUE TITLE
Error in the vendureV2Migrations script for MySql

### ISSUE THREAD

**Issue by julienTeam** [vendure-ecommerce/v2-migration-testbed#4]
Description:
I encountered a syntax error while running the vendureV2Migrations script. The error occurs in the following section of the script:

```ts
// Create a default StockLocation
const defaultStockLocationName = "Default Stock Location";
const [{id: defaultStockLocationId}] = await q(
    `INSERT INTO "stock_location" ("createdAt", "updatedAt", "name", "description") 
             VALUES (DEFAULT, DEFAULT, :defaultStockLocationName, '') RETURNING id`,
    {defaultStockLocationName}
);
console.log(
    `Created default StockLocation with id ${defaultStockLocationId}`
);
```

The error message I received is: "ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'RETURNING id' at line 2."

It seems that the RETURNING keyword is not compatible with MySQL. It is commonly used in PostgreSQL, but MySQL does not support it. Could you please provide a MySQL-compatible alternative for this query?
This could be as followed:

```ts
// Create a default StockLocation
const defaultStockLocationName = "Default Stock Location";
const [defaultStockLocation] = await queryRunner.query(
    `INSERT INTO stock_location (createdAt, updatedAt, name, description) 
    VALUES (CURRENT_TIMESTAMP(), CURRENT_TIMESTAMP(), ?, '')`,
    [defaultStockLocationName]
);

const defaultStockLocationId = defaultStockLocation.insertId;
console.log(`Created default StockLocation with id ${defaultStockLocationId}`);
```

Thank you!

**Comment by michaelbromley** [comment]
Thanks for the report! This and a couple of other MySQL issues are fixed in v0.4.0, just published.

**Comment by julienTeam** [comment]
Great! thank you,
On our side, we already succeeded the migration and will be live on V2.0.1 tomorrow morning :-)
Thank you for your work, indeed, the V2 is promising and the UI has really been improved.
